script/check_changelog.py

```diff
@@ -0,0 +1,105 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
Is this file used anywhere? Or have I missed something?

Reply:
I launch it manually for now.

Reply:
I added it as a pre-commit hook, it's fast enough.